docs/tutorials/content.md

```diff
@@ -0,0 +1,118 @@
+## ITowns' basics
+
+### 1 - Introduction
+
+### 2 - The `View` : a support for data visualization
+
+- What is a `View` ? 
+- A `View` contains the `canvas` WebGL uses to render graphics.
```

Comment:
- A View contains the Layers and a camera.
- A layer is visual representation of raster or vector map data.
- itowns supports EPSG projections. 
   - PlanarView have local conic projection (Lambert 93 EPSG:2154)
   - GlobgeView have geocentric system projection (EPSG:4978) and supports WGS84 and PM to raster projection.
- All Sources and  web services support